docs/algorithm-registry.md

```diff
@@ -15,16 +15,30 @@ specification for the design rationale for this registry.
 
 ## Signature Algorithms
 
-| Algorithm | Name                       | Usage                                             |
-|-----------|----------------------------|---------------------------------------------------|
-| RSA       | rsa-sign-pkcs1-2048-sha256 | verify only                                       |
-|           | rsa-sign-pkcs1-3072-sha256 | sign/verify                                       |
-|           | rsa-sign-pkcs1-4096-sha256 | sign/verify                                       |
-| ECDSA     | ecdsa-sha2-256-nistp256    | sign/verify                                       |
-|           | ecdsa-sha2-384-nistp384    | sign/verify                                       |
-|           | ecdsa-sha2-512-nistp521    | sign/verify                                       |
-| EdDSA     | ed25519                    | sign/verify                                       |
-|           | ed25519-ph                 | sign/verify (recommended only for `hashedrekord`) |
+| Algorithm | Name                       | Usage       | Notes                                                                            |
+|-----------|----------------------------|-------------| -------------------------------------------------------------------------------- |
+| RSA       | rsa-sign-pkcs1-2048-sha256 | verify only |                                                                                  |
+|           | rsa-sign-pkcs1-3072-sha256 | sign/verify |                                                                                  |
+|           | rsa-sign-pkcs1-4096-sha256 | sign/verify |                                                                                  |
+| ECDSA     | ecdsa-sha2-256-nistp256    | sign/verify |                                                                                  |
+|           | ecdsa-sha2-384-nistp384    | sign/verify |                                                                                  |
+|           | ecdsa-sha2-512-nistp521    | sign/verify |                                                                                  |
+| EdDSA     | ed25519                    | sign/verify |                                                                                  |
+|           | ed25519-ph                 | sign/verify | Recommended only for `hashedrekord`.                                             |
+| LMS       | lms-sha256                 | sign/verify | Stateful; signer selects the `H` parameter. Not recommended for keyless signing. |
```

Comment:
Flagging for review: I added "not recommended for keyless signing" because the chief advantage of keyless signing is the ability to use arbitrary numbers of identity-bound keys, which is the ideal scenario for LM-OTS.

Reply:
I seemed to miss the original PR that added this file and `KnownSignatureAlgorithm`, but what is the difference from the [PublicKeyDetails](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_common.proto#L52)? 

It seems a bit redundant now? And they are not really in sync either.

We did start out to separate algorithm and encoding to different messages, but decided to merge them, to avoid situation where invalid combinations was used to make it easier for clients with less failure modes.